DocumentService/DocumentService.csproj

```diff
@@ -1,11 +1,11 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
-        <TargetFrameworks>netstandard2.0;net452</TargetFrameworks>
+        <TargetFrameworks>netstandard2.0;net48</TargetFrameworks>
```

Comment:
Revert this change.

Reply:
This was done as `RuntimeInformation.IsOSPlatform` is not available in dotnet 4.5.2

https://learn.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.runtimeinformation.isosplatform?view=net-8.0